CRM/Member/Form/MembershipRenewal.php

```diff
@@ -291,6 +295,7 @@ public function buildQuickForm() {
       }
     }
 
+
```

Comment:
The changes are good and works fine for me. Would be nice if you could remove this blank line added to the patch.

Reply:
ok I removed the line.